diff --git a/dist/packagers/pnpm.js b/dist/packagers/pnpm.js
index 5ab9a53dc93751dea8584f852f13d757eeb3946f..2a8338d22bd1c6872406dbd03a843db2707b3cf3 100644
--- a/dist/packagers/pnpm.js
+++ b/dist/packagers/pnpm.js
@@ -73,9 +73,7 @@ class Pnpm {
         await (0, utils_1.spawnProcess)(command, args, { cwd });
     }
     async prune(cwd) {
-        const command = /^win/.test(process.platform) ? 'pnpm.cmd' : 'pnpm';
-        const args = ['prune'];
-        await (0, utils_1.spawnProcess)(command, args, { cwd });
+        // Prune disabled - causes lockfile mismatch with pnpm overrides
     }
     async runScripts(cwd, scriptNames) {
         const command = /^win/.test(process.platform) ? 'pnpm.cmd' : 'pnpm';
